tests/FakeItEasy.Tests/Configuration/RuleBuilderTests.cs

```diff
@@ -32,6 +34,30 @@ public RuleBuilderTests()
             this.builder = this.CreateBuilder();
         }
 
+        public static IEnumerable<object[]> CallSpecificationActionsForVoid =>
+            TestCases.FromObject<Action<IVoidArgumentValidationConfiguration>>(
+                configuration => configuration.WhenArgumentsMatch(args => true));
+
+        public static IEnumerable<object[]> CallSpecificationActionsForNonVoid =>
+            TestCases.FromObject<Action<IAnyCallConfigurationWithReturnTypeSpecified<int>>>(
+                configuration => configuration.WhenArgumentsMatch(args => true),
+                configuration => configuration.Where(call => true));
+
+        public static IEnumerable<object[]> BehaviorDefinitionActionsForVoid =>
+            TestCases.FromObject<Action<IVoidArgumentValidationConfiguration>>(
+                configuration => configuration.CallsBaseMethod(),
+                configuration => configuration.DoesNothing(),
+                configuration => configuration.Throws<Exception>(),
+                configuration => configuration.Invokes(DoNothing),

```

Comment:
This is a workaround for a StyleCop bug. I had initially written it like this:

    configuration => configuration.Invokes(_ => { }),

But StyleCop was reporting a warning:

> SA1502 : CSharp.Layout : The property must not be placed on a single line. The opening and closing curly brackets must each be placed on their own line.

There was also a warning on the following line for `new object[] { }`. Apparently StyleCop gets confused by the curly brackets in an expression-bodied property...